apps/server/src/modules/authentication/controller/dto/email-pattern.ts

```diff
@@ -0,0 +1,4 @@
+// TODO Compare with client
+export const emailPattern = new RegExp(
```

Comment:
Actually I lost that during merge probably. I changed that to be 'Matches(emailPattern)' in the entity instead of using 'isEmail'. Not sure if 'isEmail' is doing the same as the emailPattern. To play save, I'd go with the original RegEx which is also used in the client already. What's your opinion here?

Reply:
As discussed, we'll use @IsEmail()